cf/data/data.py

```diff
@@ -1748,7 +631,7 @@ def diff(self, axis=-1, n=1, inplace=False):
          [1.0 1.5 0.5]]
         >>> print(d.diff(n=2).array)
         [[0.0  0.0]
-         [ --   --]
+         [ --   --]s
```

Comment:
```suggestion
         [ --   --]
```

(Looks like an accidental keystroke or similar...)